### DISCUSSION TITLE
0.11.0-beta.1 - ENDED

### DISCUSSION THREAD

**Discussion by Taitava** [Taitava/obsidian-shellcommands#166] [Announcements]
**This is not a normal version. It's a beta test version and can be unstable.** [See these instructions, if you are willing to participate in beta testing](https://github.com/Taitava/obsidian-shellcommands/discussions/98). Do not use any important vault for testing, because you are not able to convert your data.json settings file back to an earlier version!

Most important new features:
- [Output channel: Open a file (#143)](https://github.com/Taitava/obsidian-shellcommands/issues/143).
- [Output channel _Ask after execution_: If some text is selected, use the selected text instead of the whole text (#158)](https://github.com/Taitava/obsidian-shellcommands/issues/158).

Some small features:
- [Command palette: An ability to edit the "Execute: " prefix text (#164)](https://github.com/Taitava/obsidian-shellcommands/issues/164).
- [ A global option to disable all Events (#163)](https://github.com/Taitava/obsidian-shellcommands/issues/163).

Bug fixes (nothing critical):
- [Fix a latent bug when parsing arguments in variables (#152)](https://github.com/Taitava/obsidian-shellcommands/issues/152).
- [Bug: Ghost shell commands are executed by events even after removal (#165)](https://github.com/Taitava/obsidian-shellcommands/issues/165).

<hr /><em>This discussion was created from the release <a href='https://github.com/Taitava/obsidian-shellcommands/releases/tag/0.11.0-beta.1'>0.11.0-beta.1</a>.</em>

**Comment by Taitava** [comment]
Here is documentation for the new _Open a file_ output channel: https://publish.obsidian.md/shellcommands/Output+handling/Output+channel+-+Open+a+file

**Comment by FelipeRearden** [comment]
# Version 0.11-beta.1

Hello @Taitava !!!!

## Working Perfect
#158 
#163 
#164 
Focus on Modal from here -> https://github.com/Taitava/obsidian-shellcommands/discussions/138#discussioncomment-2082029

## Question about #143 

I´m on Windows okay ......

Simple text: I have a note called Felipe.md that I created 3  months ago

I create this simple SC `Felipe.md` [output=Open a File]

Here is what happens

1- After hiiting the SC, it will open a windows message box when Windows dont have a specific app to open a file
2- I choose Obsidian.exe as default for .md files
3- After hitting the SC again, now it open the Obsidian screen to select a vault to open  

I know that  Im doing something wrong, but I dont know how :)

Have a great day!!!!

**Reply by Taitava** [reply]
Hi @FelipeRearden ! 😎 Nice to know that #158, #163 and #164 work ok for you! 🙂 

> I create this simple SC `Felipe.md` [output=Open a File]
> 
> Here is what happens
> 
> 1- After hiiting the SC, it will open a windows message box when Windows dont have a specific app to open a file 2- I choose Obsidian.exe as default for .md files 3- After hitting the SC again, now it open the Obsidian screen to select a vault to open

If I understood correctly, I think you should use `echo` command, e.g. `echo Felipe.md`. On Windows it should work both with `CMD` shell and `PowerShell`.

Offtopic: While this particular example works with `CMD` too, if you want to use variables, I'd suggest using `PowerShell` due to the [special characters escaping problem with CMD](https://publish.obsidian.md/shellcommands/Variables/Escaping+special+characters+in+variable+values#Escaping+depends+on+the+shell) I've mentioned before.

**Reply by FelipeRearden** [reply]
Thanks @Taitava !!!

I forgot about the echo sorry for the mistake :)

Let me get back to the tests and I get back to you later . I think I found an amazing workflow with another plug-in to share with you ....

By the way, the documentation is absolutely fantastic !!!!!!

**Reply by FelipeRearden** [reply]
# Version 0.11-beta.1

#143 is working perfect too !!!!!!

I tested all possibilities including `Events - File Menu` with `{{event_file_name}}` which I think will be a game changer.  

Thank you very much @Taitava !!!!!!!!

**Reply by Taitava** [reply]
Nice to know that it works! 😎 And nice to hear about new workflows and ideas! 🌞

**Comment by Taitava** [comment]
# Beta test ended

Thank you @FelipeRearden for testing and feedback! ❤️
